src/app/(test)/yj/_components/OptionWithButtons.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import { Button, Dropdown } from '@/components';
+import { DeleteIcon, MinusIcon, PlusIcon } from '@/public/index';
+import type { OptionTypes } from '@/types/ProductTypes';
+import classNames from 'classnames/bind';
+import { useState } from 'react';
+import styles from './ProductDetail.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface OptionCountProps {
+  optionText?: string;
+  price?: number;
+}
+
+interface OptionWithButtonProps {
+  optionList: OptionTypes[];
+  price: number;
+}
+
+type OperationType = 'plus' | 'minus';
+
+function OptionContainer({ optionText, price }: OptionCountProps) {
+  const [count, setCount] = useState<number>(1);
+  const [isOptionSelected, SetIsOptionSelected] = useState<boolean>(true);
+
+  const handleClickIcon = (operation: OperationType) => {
+    if (operation === 'minus' && count > 1) {
+      setCount((prev) => prev - 1);
+    } else if (operation === 'plus') {
+      setCount((prev) => prev + 1);
+    }
```

Comment:
```suggestion
    if (operation === 'minus' && count > 1) {
      setCount((prev) => prev - 1);
      return;
    } 
    
    setCount((prev) => prev + 1);
    
```
스타일 차이이긴 한데 이런식으로 많이 사용한다고 들었읍니다